package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "mymobx",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "start": "serve -s build",
+    "dev": "nodemon scripts/start.js",
+    "build": "node scripts/build.js",
+    "test": "node scripts/test.js"
+  },
+  "dependencies": {
+    "@babel/core": "^7.2.2",
```

Comment:
dependencies vs devDependencies please read about them and do corresponding changes